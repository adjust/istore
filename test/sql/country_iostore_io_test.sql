BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT 'de=>1,de=>2'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1", "de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1", "zz"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT ' "zz"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=> "+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1" ,"de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1", "de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1","de"=>"2" '::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT 'zz=>+1,de=>2 '::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:22;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT ''::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
