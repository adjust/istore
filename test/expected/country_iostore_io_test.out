BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT 'de=>1,de=>2'::country_istore;
SELECT * FROM country_istore_io;
 country_istore 
----------------
 "de"=>"3"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1", "de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
 country_istore 
----------------
 "de"=>"3"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1", "zz"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"1", "zz"=>"2"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT ' "zz"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=> "+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1" ,"de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1", "de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT '"zz"=>"+1","de"=>"2" '::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
BEGIN;
-- country_iostore_io should persist country_iostores;
-- ./spec/country_iostore_io_spec.rb:21;
CREATE EXTENSION istore;
CREATE TABLE country_istore_io AS SELECT 'zz=>+1,de=>2 '::country_istore;
SELECT * FROM country_istore_io;
    country_istore    
----------------------
 "de"=>"2", "zz"=>"1"
(1 row)

ROLLBACK;
